### PR TITLE
Add Unit tests for write_vec_to_csv and test_write_u64cols_to_file

### PR DESCRIPTION
**Pull request by jiancao-yajc**
## Types of changes

- [ ] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)
- [ ] Docs change / refactoring / dependency upgrade

## Motivation and Context / Related issue

Add Unit tests for write_vec_to_csv and test_write_u64cols_to_file
Task: [T117198333](https://www.internalfb.com/intern/tasks/?t=117198333)
## How Has This Been Tested (if it applies)

Command: cargo test
Screen Shot 2022-04-27 at 12 14 34 PM

## Checklist

- [x] The documentation is up-to-date with the changes I made.
- [x] I have read the **CONTRIBUTING** document and completed the CLA (see **CONTRIBUTING**).
- [x] All tests passed, and additional code has been covered with new tests.

### CODE REVIEW

File: common/src/files.rs [hunk 1]
```diff
@@ -259,4 +259,70 @@ mod test {
         let id_map = vec![s1, s2, s3];
         sort_stringify_id_map(&id_map, true);
     }
+
```
Comment (shubho) [inline common/src/files.rs:262]: How are these tempfiles deleteed?

**Review by shubho — CHANGES_REQUESTED** [review]

File: common/src/files.rs (hunk 1 above)
Comment (jiancao-yajc) [reply common/src/files.rs:262]: NamedTempFile relies on its destructor to do cleaning.

Comment (jiancao-yajc) [reply common/src/files.rs:262]: > NamedTempFile relies on its destructor to do cleaning.

NamedTempFile relies on its destructor to do cleaning.

**Review by shubho — CHANGES_REQUESTED** [review]

File: common/src/files.rs [hunk 2]
```diff
@@ -259,4 +259,74 @@ mod test {
...
+    }
+
+    #[test]
+    fn test_write_vec_to_csv_with_header() {
+        use tempfile::NamedTempFile;
+        use std::io::{ Read};
+        let s0 = vec![String::from("ID"),String::from("NAME")];
+        let s1 = vec![String::from("3"),String::from("a")];
+        let s2 = vec![String::from("2"),String::from("a")];
+        let s3 = vec![String::from("1"),String::from("a")];
```
[the remark this answers is not in the document]
Comment (jiancao-yajc) [reply common/src/files.rs:288]: > formatting nit: can you add space between `s0` and `s1` in line 288 below?
Done

**Review by shubho — APPROVED** [review]

**Pull request merged**